workload/src/main/java/com/google/finapp/WorkloadClient.java

```diff
@@ -0,0 +1,59 @@
+package com.google.finapp;
+
+import com.google.finapp.FinAppGrpc.FinAppBlockingStub;
+import com.google.protobuf.ByteString;
+import io.grpc.ManagedChannel;
+import io.grpc.StatusRuntimeException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class WorkloadClient {
+
+  private final ManagedChannel channel;
+  private static Logger logger = Logger.getLogger(WorkloadClient.class.getName());
```

Comment:
I need to look into how to modify this for when the workload generator is multithreaded. For now, this works.